### ISSUE TITLE
Invalid source release: 17 - Unable to build for android

### ISSUE THREAD

**Issue by pbeccegato** [itsarvinddev/barcode_scanner#99]
After last flutter upgrade, I've got the following error:

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':mobile_scanner:compileDebugJavaWithJavac'.
> error: invalid source release: 17

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 3s
Error: Gradle task assembleDebug failed with exit code 1

------ ENVIRONMENT ------

....
  ai_barcode_scanner: ^3.4.3
....

Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 3.19.5, on macOS 14.1.2 23B92 darwin-arm64, locale it-IT)
[✓] Android toolchain - develop for Android devices (Android SDK version 33.0.0-rc3)
[!] Xcode - develop for iOS and macOS (Xcode 15.3)
    ! CocoaPods 1.11.2 out of date (1.13.0 is recommended).
        CocoaPods is used to retrieve the iOS and macOS platform side's plugin code that responds to your plugin usage on the Dart side.
        Without CocoaPods, plugins will not work on iOS or macOS.
        For more info, see https://flutter.dev/platform-plugins
      To upgrade see https://guides.cocoapods.org/using/getting-started.html#updating-cocoapods for instructions.
[✓] Chrome - develop for the web
[✓] Android Studio (version 2020.3)
[✓] VS Code (version 1.87.2)
[✓] Connected device (4 available)
[✓] Network resources

! Doctor found issues in 1 category.

------

java --version
java 17.0.1 2021-10-19 LTS
Java(TM) SE Runtime Environment (build 17.0.1+12-LTS-39)
Java HotSpot(TM) 64-Bit Server VM (build 17.0.1+12-LTS-39, mixed mode, sharing)

-----

**Comment by AmitGupta3Frames** [comment]
Have you found any solution?

**Comment by pbeccegato** [comment]
> Have you found any solution?

No. I'm actually using [mobile_scanner](https://pub.dev/packages/mobile_scanner)

**Comment by rvndsngwn** [comment]
Hi @pbeccegato  and @AmitGupta3Frames, I'm working on a new version of the ai_barcode_scanner package and will release it soon.
